### PR TITLE
Add OMPS NM N20 processing

### PR DESCRIPTION
**Pull request by LouisKouvaris-NOAA**
[global_ozinfo.txt](https://github.com/NOAA-EMC/GSI/files/7021782/global_ozinfo.txt)

Added new OMPS NM QC
Added the top five layers in OMPS NP
Added monitoring OMPS NP N20

### CODE REVIEW

File: scripts/exgdas_enkf_update.sh [hunk 1]
```diff
@@ -362,6 +362,7 @@ cat > enkf.nml << EOFnml
    sattypes_oz(8) = 'mls30_aura',
    sattypes_oz(9) = 'ompsnp_npp',
    sattypes_oz(10) = 'ompstc8_npp',
+   sattypes_oz(11) = 'ompstc8_n20',
```
Comment (HaixiaLiu-NOAA) [inline scripts/exgdas_enkf_update.sh:365]: should the ompsnp_n20 be added in this script as well?

**Review by HaixiaLiu-NOAA — CHANGES_REQUESTED** [review]
I think the changes to the global_ozinfo.txt, the scripts and the setupoz.f90 look good except for the file exgdas_enkf_update.sh which should include ompsnp_n20.

**Comment by MichaelLueken-NOAA** [comment]
@LouisKouvaris-NOAA Should these changes go into the authoritative master, or to the new release/gfsda.v16.x release branch in the authoritative repo?

**Comment by MichaelLueken-NOAA** [comment]
@LouisKouvaris-NOAA This PR is currently pointing to the authoritative repo's master, not release/gfsda.v16.x.  I should be able to redirect the PR to point from master to release/gfsda.v16.x in the authoritative repo.  Please let me know if I should do this.

**Comment by LouisKouvaris-NOAA** [comment]
These changes need to go into release/gfsda.v16.x, not the master. Can you also change global_ozinfo.txt to monitor OMPS LP? Thanks.

**Comment by MichaelLueken-NOAA** [comment]
@LouisKouvaris-NOAA I have changed the branch that these changes will be merged to from the authoritative master to release/gfsda.v16.x.  However, there are now far too many changes and it looks like these changes are from the master itself.  We don't want these in the gfsda.v16.x release branch.  This PR will need to be closed and a new PR will need to be opened where the required changes are made directly to a copy of the release/gfsda.v16.x branch in your fork.

**Comment by LouisKouvaris-NOAA** [comment]
Resubmitted as PR #212

**Pull request closed without merging**